src/modules/uORB/uORBDeviceNode.hpp

```diff
@@ -195,6 +195,11 @@ class uORB::DeviceNode : public cdev::CDev, public ListNode<uORB::DeviceNode *>
 	int get_priority() const { return _priority; }
 	void set_priority(uint8_t priority) { _priority = priority; }
 
+	hrt_abstime last_update();
+
+	bool copy(void *dst, unsigned &generation);
+	uint64_t copyTime(void *dst, unsigned &generation);
```

Comment:
What about using `snake_case` and `camelCase` consistently? I'm looking at `get_priority` vs `copyTime`.

Reply:
Let's discuss this on the side. I think it's time to get a basic style guide in place (without getting too carried away).

For naming I would propose the Google style guide where simple setters and getters are named like variables (snake_case) and functions or methods are named mixed case.

Reply:
I have no opinion but I usually follow what is applied in the file just for consistency.

Reply:
Updated copyTime -> copy_time for now.